packages/styles/src/card.scss

```diff
@@ -29,73 +17,291 @@ $fd-card-header-outline-offset: 0.0625rem !default;
   text-overflow: ellipsis;
 }
 
-@mixin fake-card-outline() {
-  @include fd-focus() {
-    outline: none;
+$shell-category-colors: (
+  "1": (
+    "background": var(--sapShell_Category_1_Background),
+    "textColor": var(--sapShell_Category_1_TextColor),
+    "textShadow": var(--sapShell_Category_1_TextShadow)
+  ),
+  "2": (
+    "background": var(--sapShell_Category_2_Background),
+    "textColor": var(--sapShell_Category_2_TextColor),
+    "textShadow": var(--sapShell_Category_2_TextShadow)
+  ),
+  "3": (
+    "background": var(--sapShell_Category_3_Background),
+    "textColor": var(--sapShell_Category_3_TextColor),
+    "textShadow": var(--sapShell_Category_3_TextShadow)
+  ),
+  "4": (
+    "background": var(--sapShell_Category_4_Background),
+    "textColor": var(--sapShell_Category_4_TextColor),
+    "textShadow": var(--sapShell_Category_4_TextShadow)
+  ),
+  "5": (
+    "background": var(--sapShell_Category_5_Background),
+    "textColor": var(--sapShell_Category_5_TextColor),
+    "textShadow": var(--sapShell_Category_5_TextShadow)
+  ),
+  "6": (
+    "background": var(--sapShell_Category_6_Background),
+    "textColor": var(--sapShell_Category_6_TextColor),
+    "textShadow": var(--sapShell_Category_6_TextShadow)
+  ),
+  "7": (
+    "background": var(--sapShell_Category_7_Background),
+    "textColor": var(--sapShell_Category_7_TextColor),
+    "textShadow": var(--sapShell_Category_7_TextShadow)
+  ),
+  "8": (
+    "background": var(--sapShell_Category_8_Background),
+    "textColor": var(--sapShell_Category_8_TextColor),
+    "textShadow": var(--sapShell_Category_8_TextShadow)
+  ),
+  "9": (
+    "background": var(--sapShell_Category_9_Background),
+    "textColor": var(--sapShell_Category_9_TextColor),
+    "textShadow": var(--sapShell_Category_9_TextShadow)
+  ),
+  "10": (
+    "background": var(--sapShell_Category_10_Background),
+    "textColor": var(--sapShell_Category_10_TextColor),
+    "textShadow": var(--sapShell_Category_10_TextShadow)
+  ),
+  "11": (
+    "background": var(--sapShell_Category_11_Background),
+    "textColor": var(--sapShell_Category_11_TextColor),
+    "textShadow": var(--sapShell_Category_11_TextShadow)
+  ),
+  "12": (
+    "background": var(--sapShell_Category_12_Background),
+    "textColor": var(--sapShell_Category_12_TextColor),
+    "textShadow": var(--sapShell_Category_12_TextShadow)
+  ),
+  "13": (
+    "background": var(--sapShell_Category_13_Background),
+    "textColor": var(--sapShell_Category_13_TextColor),
+    "textShadow": var(--sapShell_Category_13_TextShadow)
+  ),
+  "14": (
+    "background": var(--sapShell_Category_14_Background),
+    "textColor": var(--sapShell_Category_14_TextColor),
+    "textShadow": var(--sapShell_Category_14_TextShadow)
+  ),
+  "15": (
+    "background": var(--sapShell_Category_15_Background),
+    "textColor": var(--sapShell_Category_15_TextColor),
+    "textShadow": var(--sapShell_Category_15_TextShadow)
+  ),
+  "16": (
```

Comment:
Maybe @for would be nice here? https://sass-lang.com/documentation/at-rules/control/for/